app/addons/dashboard/resources.js

```diff
@@ -12,9 +12,10 @@
 
 define([
   'app',
-  'api'
+  'api',
+  'addons/databases/resources'
 ],
-function (app, FauxtonAPI) {
+function (app, FauxtonAPI, Databases) {
```

Comment:
you don't really need this file, you can take it (addons/databases/resources) out